Font management

gibtmirdas opened the pull request:
Add possibility to specify a ttf font path whenever a message is written on the display

shawaj commented:
Thanks very much. This looks like a great little contribution. Have you tested and verified it works as expected?

@gibtmirdas can you draft up a little text for us to use in the readme describing how to use this? Can just post it here and we will add it to the readme

Thanks

gibtmirdas commented:
Hi,

I added some documentation in the README.md, I hope it's fine for you.
You were right about textpos:127. I fixed it, thank you.

I tested it and everything seems to work fine.

Thanks for your library !

shawaj commented:
Thanks very much. We will perform some tests next week and then merge if all ok.

Thanks
